### ISSUE TITLE
Require that the user accepts terms & conditions before starting to use the app

### ISSUE THREAD

**Issue by pyritewolf** [instedd/oasis#9]
Terms & conditions are, right now, optional. They should be available during the sign up process, and accepting them should be mandatory to proceed with sign-up

**Comment by pyritewolf** [comment]
@ste519 hello there! I just realized this might be a good issue for you if you happen to have the time! 😄

**Comment by easpencer** [comment]
Updated and edited TC is here:
https://docs.google.com/document/d/1RrGjm1vxn2LaZj2WxpDOgQPdphHHNXVCkqolpSXSbFE

@ste519 
@pyritewolf

**Comment by ste519** [comment]
Duplicate of #173
